GameFighterXUnitTests/GameFighterTests.cs

```diff
@@ -0,0 +1,175 @@
+using GameFighter;
+using System;
+using Xunit;
+
+namespace GameFighterXUnitTests
+{
+    public class GameFighterTests
+    {
+        [Fact]
+        public void Fight_TwoWarriors_ReturnTrue()
+        {
+            var warrior1 = new Warrior();
+            var warrior2 = new Warrior();
+
+            Assert.True(Battle.Fight(warrior1, warrior2));
+        }
+
+        [Fact]
+        public void Fight_WarriorVsKnight_ReturnFalse()
+        {
+            var warrior = new Warrior();
+            var knight = new Knight();
+
+            Assert.False(Battle.Fight(warrior, knight));
+        }
+
+        [Fact]
+        public void Fight_KnightVsWarrior_ReturnTrue()
+        {
+            var warrior = new Warrior();
+            var knight = new Knight();
+
+            Assert.True(Battle.Fight(knight, warrior));
+        }
+
+        [Fact]
+        public void Fight_KnightVsKnight_ReternTrue()
+        {
+            var knight1 = new Knight();
+            var knight2 = new Knight();
+
+            Assert.True(Battle.Fight(knight1, knight2));
+        }
+
+        [Fact]
+        public void Fight_IsWarrior1AfterBattleVsWarriorAlive_ReternTrue()
```

Comment:
I would rename this test to Fight_WarriorVsWarrior_SecondWarriorIsAlive. You don't need to ask in the test name, you should use statements